client/go/vespa/target.go

```diff
@@ -129,7 +128,7 @@ func (s *Service) Description() string {
 
 func (t *customTarget) Type() string { return t.targetType }
 
-func (t *customTarget) Service(name string, timeout time.Duration, sessionID int64) (*Service, error) {
+func (t *customTarget) Service(name string, timeout time.Duration, _ int64, _ string) (*Service, error) {
```

Comment:
I think it's better to keep the parameter names the same as the interface declaration, even if they're unused.

Reply:
OK.